### DISCUSSION TITLE
Hello all!

### DISCUSSION THREAD

**Discussion by antisocrates** [community/community#35354] [GitHub Education]
Just joined github education. I strongly suggest you try out the jetbrains product line that comes free with the pack!

**Comment by samus-aran** [comment]
👋 Welcome to the GitHub Community on Discussions, @antisocrates ! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776) and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
